hack/gofmt.sh

```diff
@@ -16,7 +16,8 @@ if [ "${IS_CONTAINER}" != "false" ]; then
   # File length to be checked should be 2 since the following headers should be present
```

Comment:
nit: I think it should be 3
```suggestion
  # File length to be checked should be 3 since the following headers should be present
```

Reply:
Thanks, fixed